jetty-servlet/src/main/java/module-info.java

```diff
@@ -27,4 +27,6 @@
     requires static java.desktop;
     // Only required if using JMX.
     requires static org.eclipse.jetty.jmx;
+    requires jetty.servlet.api;
```

Comment:
@sbordet same again here:  I don't see why Eclipse has added these 2 as jetty-servlet already uses classes from the api and from util.

Reply:
Not necessary, remove it.